### DISCUSSION TITLE
Xarrow pro version and  features

### DISCUSSION THREAD

**Discussion by lucaSyn** [Eliav2/react-xarrows#98] [Q&A] [answered]
Hi, 
I started implementing your library and was searching for a way to draw arrows around elements in their way.
I found this thread in githubmemories: https://githubmemory.com/repo/Eliav2/react-xarrows/issues/70
It is exactly what me and my company are looking for. 
Is this feature coming to the open react-xarrows library or will it stay solely available in the pro version?

**Comment by Eliav2** [comment, marked as answer]
both the open-source and the pro versions are being worked on. the pro version would include a sophisticated algorithm that could avoid given elements in the path and some extra simplifications for an even better developer experience (like an auto-normalized custom SVG shapes, currently you have to normalize your SVG's your self) and more
